scipy/special/_logsumexp.py

```diff
@@ -141,9 +141,13 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
     return (out, sgn) if return_sign else out
 
 
-def _wrap_radians(x):
+def _wrap_radians(x, xp):
     # Wrap radians to -pi, pi interval
-    return (x + math.pi) % (2 * math.pi) - math.pi
+    out = (x + math.pi) % (2 * math.pi) - math.pi
```

Comment:
Currently, this produces output in $[-\pi, \pi)$.
With the change in the last commit, it would be $[-\pi, \pi]$.
If we want $(-\pi, \pi]$, this should be:
```suggestion
    out = -((-x + math.pi) % (2 * math.pi) - math.pi)
```